sdk/core/core-lro/src/poller.ts

```diff
@@ -25,28 +44,31 @@ export class PollerCancelledError extends Error {
 }
 
 /**
- * Abstract representation of a poller, intended to expose just the minimal that the user needs to work with.
+ * Abstract representation of a poller, intended to expose just the minimal API that the user needs to work with.
  */
 export interface PollerLike<TState extends PollOperationState<TResult>, TResult> {
   /**
-   * Defines under what conditions to reach out to the underlying service.
-   * It should call the operation's update method.
+   * Returns a promise that will resolve once a single polling request finishes.
+   * It does this by calling the update method of the Poller's operation.
    */
   poll(options?: { abortSignal?: AbortSignal }): Promise<void>;
```

Comment:
shouldn't this take an `AbortSignalLike` instead of an `AbortSignal`?

Reply:
Thanks!